backend/tracim_backend/lib/mail_notifier/utils.py

```diff
@@ -24,3 +37,65 @@ class EST(object):
     @classmethod
     def all(cls):
         return [cls.CONTENT_LABEL, cls.CONTENT_STATUS_LABEL, cls.WEBSITE_TITLE, cls.WORKSPACE_LABEL]
+
+
+class EmailAddress(object):
+    def __init__(self, label: str, email: str, force_angle_bracket=False):
+        self.label = label
+        self.email = email
+        self.force_angle_bracket = force_angle_bracket
+
+    @classmethod
+    def from_rfc_email_address(cls, rfc_email: str) -> "EmailAddress":
+        label, email = parseaddr(rfc_email)
+        return EmailAddress(label, email)
+
+    @property
+    def address(self):
+        formatted_address = formataddr((self.label, self.email))
+        if not self.force_angle_bracket or self.label:
+            return formatted_address
+        else:
```

Comment:
You may get rid of this "else:" if you wish

Reply:
done